spec/factories/courses.rb

```diff
@@ -23,11 +23,45 @@
 
 FactoryBot.define do
   factory :course do
+    trait :in_further_education do
```

Comment:
Maybe just fussy but I liked it more when fields was at the top, and the traits was at the bottom

Reply:
The factory is using a trait that's defined further up...

Reply:
```ruby 
Factory.define
  factory :course
#those fields
    sequence(:course_code) { |n| "C#{n}D3" }
    name { Faker::ProgrammingLanguage.name }
    qualification { 1 }
    association(:provider)
    resulting_in_pgce_with_qts

    transient do
      age { nil }
end

# lots of traits here
```
vs


```ruby 
Factory.define
  factory :course

# lots of traits here

#those fields
    sequence(:course_code) { |n| "C#{n}D3" }
    name { Faker::ProgrammingLanguage.name }
    qualification { 1 }
    association(:provider)
    resulting_in_pgce_with_qts

    transient do
      age { nil }
end

```


That was what I meant

Reply:
I got what you meant, but I didn't think that it would work to include a trait in a factory above the definition of the trait. However having tested it, it seems to be fine so I moved the traits to the bottom of the file and force-pushed.